pkg/controller/configmap/configmap_controller.go

```diff
@@ -115,6 +115,7 @@ func (r *ReconcileConfigMap) Reconcile(request reconcile.Request) (reconcile.Res
 			operatortypes.OperatorNamespace))
 		watchedNamespace = operatortypes.OperatorNamespace
 	}
+    log.Info(fmt.Sprintf("configmap controller watching events in namespace %s", watchedNamespace))
```

Comment:
add 2 more white spaces

Reply:
Done - thank you! I always forget to open vim with my golang setting and use the python settings....